requirements/requirements-dev.in

```diff
@@ -7,7 +7,7 @@
 -r requirements-docs.txt
 
 # Dependencies
-pip-tools~=3.8.0  # this is used by 'deps.py'
+pip-tools~=4.4.0  # this is used by 'deps.py'
```

Comment:
This isn't strictly necessary. But in this case it's probably a good idea to not allow 'accidental' upgrades.

Reply:
Yeah, it was pinned before, so I didn't change it. But I can unpin it if you like.

Reply:
Nah it's fine.